pydantic_ai_slim/pydantic_ai/models/anthropic.py

```diff
@@ -49,7 +49,7 @@
     'max_tokens': 'length',
     'stop_sequence': 'stop',
     'tool_use': 'tool_call',
-    'pause_turn': 'stop',
+    'pause_turn': 'stop',  # TODO: should this be a different finish reason?
```

Comment:
I think we can remove it here because we never end up returning a `ModelResponse` from `pause_turn`, right? As we always resubmit the request